packages/openclaw-plugin/ui/src/styles.css

```
@@ -1702,3 +1702,248 @@ pre {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  max-width: 180px;
+  display: block;
+}
+
+/* Code blocks for trigger / anti-pattern */
+.code-block {
+  font-size: var(--text-base);
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)